game.py

```diff
@@ -500,10 +500,11 @@ def main():
     # Variable to keep our game loop running
     running = True
     clock = pygame.time.Clock()
-    # n = Network()
+    n = Network()
     # startP = n.get_p()
 
     p1 = Player(200, 200, 40, 60, (0, 0, 255))
+    p1_server_data_resp = n.get_p()
```

Comment:
Going to change this in future to probably just be a string that just says "Player 1 Client" so that the server can check it when it first sends over so that it KNOWS that it is the game client